blocks/links-bar-block/package.json

```diff
@@ -8,10 +8,11 @@
   ],
```

Comment:
can you also update the author to be based on the checklist. and remove the contributors. Let's keep on using the checklist for updating the blocks https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3106243821/Converting+a+v1+block+to+v2+architecture